### ISSUE TITLE
update documentation building workflow to mkdocs

### ISSUE THREAD

**Issue by DaniBodor** [EIT-ALIVE/eitprocessing#254]
I wanted to create a workflow that checks whether the documentation can be built without errors. This issue might be a duplicate of #252 though.

blocked by #233

**Comment by psomhorst** [comment]
This has not been implemented with the new documentation building workflow. I created a new PR #318 to deal with this.

**Comment by DaniBodor** [comment]
I believe this was resolved in #318
